roles/falcon_configure/meta/main.yml

```diff
@@ -30,31 +30,32 @@ galaxy_info:
   platforms:
     - name: Amazon Linux 2
       versions:
-        - any
-    - name: Amazon Linux
+        - all
+    - name: Amazon
       versions:
-        - 2018.03
-        - 2017.09
+        - "2018.03"
+        - "2017.09"
     - name: Debian
       versions:
-        - 9
-        - 10
+        - "stretch"
+        - "buster"
+        - "bullseye"
     - name: EL
       versions:
-        - 6
-        - 7
-        - 8
+        - "6"
+        - "7"
+        - "8"
     - name: SLES
       versions:
-        - 15
-        - 12SP1
-        - 12
-        - 11SP4
+        - "15"
+        - "12SP1"
+        - "12"
+        - "11SP4"
     - name: Ubuntu
       versions:
-        - focal
-        - bionic
-        - xenial
+        - "focal"
```

Comment:
Really? Ansible requires to quote strings even though it is valid yaml?

Reply:
Yelled at me for everything man lol. I'm tempted to let this pr sit a few days in the event that more people start having these issues as well. Thoughts?

Reply:
hahahahaha... yeah probably should.